### DISCUSSION TITLE
bypy list 找不到任何文件

### DISCUSSION THREAD

**Discussion by holyman2k** [houtianze/bypy#632] [Q&A]
bypy list 找不到任何文件。  

只有 /apps/bypy ($t $f $s $m $d):

根目录下有文件 比如photo

如果输入 bypy list photo 会出错

<E> [21:01:12] ----------------------------------------------------------------
Error 31066

bypy quota   和 bypy whoami 都工作

求帮助

**Comment by qq254963746** [comment]
+1

**Comment by hyz159** [comment]
这个显示的是网盘里的我的应用数据/bypy下的文件

**Comment by dev-yze** [comment]
不要输入/apps/bypy这个前缀

**Reply by wangxu627** [reply]
你的bypy还能下载吗？我今天用一只报错，之前是可以的

**Reply by dev-yze** [reply]
可以

**Reply by dev-yze** [reply]
一直在用，速度每秒5M以上

**Comment by liuzan-info** [comment]
> 一直在用，速度每秒5M以上

现在还可以用吗？这两天一直报错，无法正常使用
